commands/credentials.js

```diff
@@ -9,11 +9,25 @@ function * run (context, heroku) {
   const {app, args, flags} = context
 
   let showCredentials = co.wrap(function * () {
-    let db = yield fetcher.database(app, args.database)
-    cli.log(`Connection info string:
+    const host = require('../lib/host')
+    let addon = yield fetcher.addon(app, args.database)
+    try {
+      let credentials = yield heroku.get(`/postgres/v0/databases/${addon.name}/credentials`,
+                                       { host: host(addon) })
+      cli.table(credentials, {
+        columns: [
+          {key: 'name', label: 'Credential'},
+          {key: 'state', label: 'State'}
+        ]
+      })
+    } catch (err) {
```

Comment:
So we can't do much about not supporting `--reset` here for the new interface, right? We can drop it once the feature is public?